app/components/Root/IntlWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react'
+import { IntlProvider } from 'react-intl'
+
+import translations from '../../translations'
+
+const { english } = translations
+
+type IntlWrapperProps = {
+  children: any,
+  lang: string,
+}
+
+class IntlWrapper extends React.Component<IntlWrapperProps> {
+  render() {
+    const { children, lang } = this.props
+    return (
+      <IntlProvider locale="en" messages={english}>
+        {children}
+      </IntlProvider>
+    )
+  }
+}
+
+export default IntlWrapper
```

Comment:
The `IntlWrapper` component will allow us to inject the language desired into the application dynamically via props from our redux implementation. (Right now hard coded to use "en")

Reply:
NOTE: updated to consume language from global state and persisted settings